### ISSUE TITLE
Segmentation fault in continue training inception-v3

### ISSUE THREAD

**Issue by wranglerwong** [apache/incubator-mxnet#1577]
Last month i trained inception v3, but the validation accuracy is not good when it reached 100 epoch, so  i stoped training.

Yesterday i continued training of pre-trained v3 model "inception-v3-0-0100.params", but i got this error message:
# ================================================================================

[15:58:57] src/io/iter_image_recordio.cc:212: ImageRecordIOParser: ../recordIO/train_shuffle.rec, use 3 threads for decoding..
[15:58:58] src/io/iter_image_recordio.cc:212: ImageRecordIOParser: ../recordIO/val1000c.rec, use 3 threads for decoding..
cmd.sh: line 15: 27836 **Segmentation fault**      (core dumped) python run.py --network=$NETWORK --load-epoch=$LOAD_EPOCH --data-dir=$DATA_DIR --model-prefix=$MODEL_PREFIX --log-dir=$LOG_DIR --log-file=$LOG_FILE --lr=$LR --lr-factor=$LR_FACTOR --lr-factor-epoch=$LR_FACTOR_EPOCH --clip-gradient=$CLIP_GRADIENT --num-epochs=$NUM_EPOCHES --batch-size=$BATCH_SIZE --gpus=$GPUS
# ================================================================================

I only added **--load-epoch=100**

the code of loading model:
          tmp = mx.model.FeedForward.load(model_prefix, args.load_epoch)
          model_args = {'arg_params' : tmp.arg_params,
                                  'aux_params' : tmp.aux_params,
                                  'begin_epoch' : args.load_epoch} 

How can I solve it?
Any input appreciated, thanks and best!

**Comment by piiswrong** [comment]
Could you use gdb and backtrace to locate the point of crash?

**Comment by AliceeXu** [comment]
I meet the same problem when i try to retrain inception-v3, is this problem be solved now?

**Comment by phunterlau** [comment]
This issue is closed due to lack of activity in the last 90 days. Feel free to reopen if this is still an active issue. Thanks!
